netci.groovy

```diff
@@ -754,8 +754,13 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
                 case 'Ubuntu':
                     switch(architecture) {
                         case "arm":
-                            // Removing the regex will cause this to run on each PR.
-                            Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} Cross ${configuration} Build")
+                            if (!isLinuxEmulatorBuild) {
+                                // Removing the regex will cause this to run on each PR.
+                                Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} Cross ${configuration} Build")
+                            }
+                            else {
+                                Utilities.addGithubPRTriggerForBranch(job, branch, "Linux ARM Emulator Build")
```

Comment:
Can we add a trigger phrase to make this non-default initially, work out whatever kinks there are and then enable by default for each PR?  This is just to avoid a case where it might be initially broken (say we discover we need to alter the setup of the Ubuntu image).  You can see the other calls to this function which have an extra parameter (the trigger phrase).  That will make the job non-default.